.github/workflows/pull-request.yml

```
@@ -2,7 +2,8 @@ name: Pull Request CI
 
 on:
   pull_request:
-    branches: [ main, "release/**" ]
+    branches: [ main, "release/**", "develop", "**-dev", "**-review" ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)